src/Settings/components/language.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import styles from "../styles";
+import { setLanguage, LANGUAGES } from "../../common/actions";
+import { useSelector } from "react-redux";
```

Comment:
It is the best practice to import all the 3rd party imports first and then internal ones, So I think, { useSelector } should go to line no. 5 here.

Reply:
Fixed